ObsoleteModelica4.mo

```diff
@@ -2897,7 +2897,7 @@ In most cases, this means that a model with the name
 and then a manual adaptation is needed. For example, a reference to
 ObsoleteModelica4.Math.Matrices.LAPACK.dgeqpf
 should be replaced by
-Modelica.Math.Matrices.LAPACK.dgeqp3 (version 4.0.0).
+Modelica.Math.Matrices.LAPACK.dgeqp3 (version >= 4.0.0).
```

Comment:
See @maltelenz's comment above.  I suggest simply dropping the version here:
```suggestion
Modelica.Math.Matrices.LAPACK.dgeqp3.
```
and additionally change line 2896 above like this:
```
\"ObsoleteModelica4.XXX\" should be renamed to \"Modelica.YYY\".
```
Note that there should be no need to point of the version of MSL used here, since ObsoleteModelica4 doesn't exist for versions prior to 4.